zipline/finance/trading.py

```diff
@@ -93,13 +92,34 @@ def __init__(
         max_date=None,
         env_trading_calendar=tradingcalendar
     ):
+        # `tc_tdays` is short for "trading calendar trading days"
+        # `tc_tday` is short for "trading calendar trading day"
+        tc_tdays = env_trading_calendar.trading_days
+        tc_tday = env_trading_calendar.trading_day
```

Comment:
trading_days is a pandas date range and trading_day is declared using [Custom Business Day](http://pandas.pydata.org/pandas-docs/version/0.12.0/timeseries.html#custom-business-days-experimental)  as follows
```
trading_day = pd.tseries.offsets.CDay(holidays=non_trading_days)
```